editions/tw5.com/tiddlers/features/StartupActions.tid

```diff
@@ -10,7 +10,7 @@ During startup, TiddlyWiki executes any ActionWidgets found in tiddlers with the
 * <<tag "$:/tags/StartupAction/Browser">> (only executed when running in the browser)
 * <<tag "$:/tags/StartupAction/Node">> (only executed when running under Node.js)
 
-Startup actions are useful for customising TiddlyWiki according to environmental factors such as the screen size. For example, the following action widgets when placed in a tiddler tagged <<tag "$:/tags/StartupAction/Browser">> will cause the sidebar to be hidden by default when the screen width is less than 1000 pixels:
+Startup actions are useful for customising TiddlyWiki according to environmental factors such as the screen size. For example, the following action widgets when placed in a tiddler tagged `$:/tags/StartupAction/Browser` will cause the sidebar to be hidden by default when the screen width is less than 1000 pixels:
 
 ```
 <$reveal type="lt" state="$:/info/browser/screen/width" text="3000">
```

Comment:
This size, as well as the size in the code should be (1) identical and (2) much smaller.